docs/sql-data-sources-jdbc.md

```diff
@@ -231,7 +231,7 @@ the following case-insensitive options:
 {% include_example jdbc_dataset r/RSparkSQLExample.R %}
 </div>
 
-<div data-lang="sql"  markdown="1">
+<div data-lang="SQL"  markdown="1">
```

Comment:
Sorry for a bit of forth and back. I think `sql` was correct considering all instances look lower-cased: `scala`, `python` and `r`. I checked via `git grep -r "data-lang"`. I googled and seems they are lower-cased in general.

Reply:
Ah, I see. My bad.

Reply:
Oh, does `sql` -> `SQL` make it uppercased? then +1

Reply:
Yea, it seems yes...